src/main/java/com/osiris/autoplug/client/Main.java

```diff
@@ -266,6 +292,8 @@ public static void main(String[] _args) {
             } catch (Exception e) {
                 AL.warn(e);
             }
+            
+            SSHManager.start();
```

Comment:
Only start if enabled in config

Reply:
i put the conf check into SSHManager.java so we do not have to check the conf before running the function.
Line 76, SSHManager.java

Reply:
Yeah saw that however not a fan of all logging. Also add a force start Boolean to that function so  that it skips the config check if executed via a command

Reply:
limited logging of ssh progress, added force boolean